backend/src/core/services/upload.service.ts

```diff
@@ -14,6 +14,13 @@ import { Campaign } from '@core/models'
 import { CsvStatusInterface } from '@core/interfaces'
 import { CSVParams } from '@core/types'
 
+import {
```

Comment:
Not sure if this is a good pattern, was also considering creating a new service called `MultiUploadService`